bitwarden_license/src/Commercial.Core/SecretsManager/Commands/Secrets/DeleteSecretCommand.cs

```diff
@@ -10,25 +11,28 @@ public class DeleteSecretCommand : IDeleteSecretCommand
 {
     private readonly ICurrentContext _currentContext;
     private readonly ISecretRepository _secretRepository;
+    private readonly IProjectRepository _projectRepository;
 
-    public DeleteSecretCommand(ICurrentContext currentContext, ISecretRepository secretRepository)
+    public DeleteSecretCommand(ISecretRepository secretRepository, IProjectRepository projectRepository, ICurrentContext currentContext)
     {
         _currentContext = currentContext;
         _secretRepository = secretRepository;
+        _projectRepository = projectRepository;
+        _currentContext = currentContext;
```

Comment:
`_currentContext = currentContext;`

We have this statement twice.

Remove duplicate assignment